src/sections/product.liquid

```diff
@@ -36,40 +36,53 @@
   <p>{{ product.vendor }}</p>
 
 
-  <form action="/cart/add" method="post" enctype="multipart/form-data">
+  {% form "product", product, data-product-form: '', data-product-handle:product.handle, data-enable-history-state: 'true' %}
     {% unless product.has_only_default_variant %}
       {% for option in product.options_with_values %}
-        <div class="selector-wrapper js">
-          <label for="SingleOptionSelector-{{ forloop.index0 }}">
-            {{ option.name }}
-          </label>
-
-          <select
-            id="SingleOptionSelector-{{ forloop.index0 }}"
-            data-single-option-selector
-            data-index="option{{ option.position }}">
+        <div class="js">
+
+          {% if forloop.index0 > 0 %}
```

Comment:
Why are we only allowing one dropdown element now?

Reply:
Just have it here to demonstrate you can use select + radio buttons.